docs/UI/OnDriverDistraction/index.md

```diff
@@ -11,6 +11,18 @@ Purpose
 
 ### Notification
 
+!!! must
+
+1. Inform SDL whenever the driver distraction mode is activated/deactivated on HMI.
+
+!!!
+
+Driver distraction rules may be specific to contry/area, so it depends on HMI when to trigger activate/deactivate states.
```

Comment:
`contry` -> `country`

Reply:
thank you, fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/137/commits/3efc9161566c0c35b6c4bdaf00827bd0914895d0